groupnews-web/src/test/java/com/epam/internship/ExcpetionControllerTest.java

```diff
@@ -1,47 +0,0 @@
-package com.epam.internship;
-
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.mockito.Mockito;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
-import org.springframework.boot.test.mock.mockito.MockBean;
-import org.springframework.dao.DataIntegrityViolationException;
-import org.springframework.test.context.junit4.SpringRunner;
-import org.springframework.test.web.servlet.MockMvc;
-import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
-import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
-
-import com.epam.internship.controller.UserController;
-
-@RunWith(SpringRunner.class)
-@WebMvcTest(controllers = UserController.class)
-public class ExcpetionControllerTest {
```

Comment:
Because those cases were testing the UserController anyway so migrated them to UserControllerTest class.